src/anyio/_backends/_asyncio.py

```diff
@@ -726,12 +711,28 @@ async def run_sync_in_worker_thread(
             future: asyncio.Future = asyncio.Future()
             work_queue.put_nowait((func, args, future))
             if not idle_workers:
-                args = (work_queue, workers, idle_workers)
-                thread = Thread(target=_thread_pool_worker, args=args, name='AnyIO worker thread')
+                thread = Thread(target=_thread_pool_worker, args=(work_queue, get_running_loop()),
+                                name='AnyIO worker thread')
                 workers.add(thread)
                 thread.start()
+            else:
+                thread, idle_since = idle_workers.pop()
```

Comment:
`idle_since` seems unused.  Is that intentional?

Reply:
Well, it does come out when I `pop()` from `idle_workers`, and I figured this serves as ad-hoc documentation of _what_ comes out. Doesn't matter either way I guess?

Reply:
OK that makes sense, we only care about the `idle_since` value of workers remaining.